engine/api/project/dao_dependencies.go

```diff
@@ -184,6 +184,15 @@ var (
 		return nil
 	}
 
+	loadIcon = func(db gorp.SqlExecutor, store cache.Store, proj *sdk.Project, u *sdk.User) error {
```

Comment:
shouldn't it be better to load the icon directly to avoid executing an additional query ?

Reply:
I think no, we load everywhere project, and icon could be 100ko and we just have to fetch for UI